features/step_definitions/flipkart.rb

```diff
@@ -0,0 +1,44 @@
+require 'selenium-webdriver'
+
+  caps = {
+  	'bstack:options' => {
+  		"os" => "Windows",
+  		"osVersion" => "10",
+  		"local" => "false",
+  		"consoleLogs" => "info",
+  		"networkLogs" => "true",
+  		"seleniumVersion" => "3.5.2",
+  	},
+  	"browserName" => "IE",
+  	"browserVersion" => "11.0",
+  }
+
+
+
+  driver = Selenium::WebDriver.for(:remote,
+    :url => "http://prateekbhiwapurk2:5yDG4MWRdXoFx8Fzg2Mi@hub-cloud.browserstack.com/wd/hub",
+    :desired_capabilities => caps)
```

Comment:
Keys are sensitive use as an environment variable

Reply:
Also recycle your key on browserstack

Reply:
Used ENV['USERNAME'] and ENV['PASSWORD'] and passing these through command line now